src/main/java/com/cinema/model/dto/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,11 @@
+package com.cinema.model.dto;
+
+import java.util.List;
+import lombok.Data;
+
+@Data
+public class ShoppingCartResponseDto {
+    private Long id;
+    private Long userId;
```

Comment:
these two have the same value, right?

Reply:
Yeah, they should be identical, you're right!